tests/xbot/common/injection/UnitTestModule.java

```diff
@@ -27,7 +32,13 @@ protected void configure() {
         this.bind(WPIFactory.class).to(MockWPIFactory.class);
 
         this.bind(ITableProxy.class).to(TableProxy.class).in(Singleton.class);
-        this.bind(DatabaseStorageBase.class).to(OffRobotDatabaseStorage.class).in(Singleton.class);
+        
+        if(this.useRealDatabaseForPropertyStorage) {
```

Comment:
We should use a variable here to store the target class instead of duplicating the rest of the line